src/Nethermind/Nethermind.Specs/ChainSpecStyle/Json/ChainSpecParamsJson.cs

```diff
@@ -145,6 +145,8 @@ internal class ChainSpecParamsJson
 
         public long? TerminalPoWBlockNumber { get; set; }
 
-        public long? Eip1153Transition { get; set; }
+        public ulong? Eip1153TransitionTimestamp { get; set; }
```

Comment:
And again we should add other EIPs :)

Reply:
Will be added with their PRs or by merging shandong